### PR TITLE
Change default value to true for transpositions parameter of fuzzy query

### PR DESCRIPTION
**Pull request by alexshadow007**
All other fuzzy queries (`match`, `multi_match`, `query_string` and `simple_query_string`) are using `true` as default value for `transpositions` parameter.

### CODE REVIEW

File: core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java [hunk 1]
```diff
@@ -54,8 +54,8 @@
     public static final int DEFAULT_MAX_EXPANSIONS = FuzzyQuery.defaultMaxExpansions;
 
     /** Default as to whether transpositions should be treated as a primitive edit operation,
-     * instead of classic Levenshtein algorithm. Defaults to false. */
-    public static final boolean DEFAULT_TRANSPOSITIONS = false;
+     * instead of classic Levenshtein algorithm. Defaults to true. */
+    public static final boolean DEFAULT_TRANSPOSITIONS = FuzzyQuery.defaultTranspositions;
```
Comment (jimczi) [inline core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java]: Can you add a small test in `FuzzyQueryBuilderTests` that setting `transpositions` to true/false works ?
You can also remove the comment below:
````
//LUCENE 4 UPGRADE ...
````

**Review by jimczi — CHANGES_REQUESTED** [review]
I left a comment to add a test.
Thanks @alexshadow007 !

**Comment by alexshadow007** [comment]
@jimczi Done

**Review by jimczi — APPROVED** [review]
Thanks @alexshadow007 !

**Comment by jimczi** [comment]
@elasticmachine ok to test

**Pull request merged**